atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -81,24 +81,12 @@ def run(self):
         else:
             self.log.info("V2 schema 2 digest is not available")
 
-        while True:
-            # Pull the image from Crane
-            name = self.tasker.pull_image(pullspec, insecure=self.insecure)
-
-            # Inspect it
-            try:
-                metadata = self.tasker.inspect_image(name)
-            except NotFound:
-                if time() - start > self.timeout:
-                    raise CraneTimeoutError("{} seconds exceeded"
-                                            .format(self.timeout))
-
-                self.log.info("will try again in %ss", self.retry_delay)
-                sleep(self.retry_delay)
-                continue
-
-            defer_removal(self.workflow, name)
-            break
+        # Pull the image from Crane
+        name = self.tasker.pull_image(pullspec, insecure=self.insecure)
```

Comment:
I think this ought to perform special handling for not_found, since it "knows" that it expects the content to appear at a later time.